jena-arq/src/main/java/org/apache/jena/sparql/algebra/OpVisitor.java

```diff
@@ -23,53 +23,53 @@
 public interface OpVisitor
 {
     // Op0
-    public void visit(OpBGP opBGP) ;
-    public void visit(OpQuadPattern quadPattern) ;
-    public void visit(OpQuadBlock quadBlock) ;
-    public void visit(OpTriple opTriple) ;
-    public void visit(OpQuad opQuad) ;
-    public void visit(OpPath opPath) ;
-    public void visit(OpTable opTable) ;
-    public void visit(OpNull opNull) ;
+    default void visit(OpBGP opBGP) {}
```

Comment:
I figured the general implementation of `visit` is NOOP, but I will pull this out.